### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RuslanMelnykFE**

### CODE REVIEW

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Code looks interesting but demo link doesn't work, try to fix it and put this link to your pull request description

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,472 @@
 'use strict';
 
-// write your code here
+//==============================================================
```
Comment (TarasHoliuk) [inline src/scripts/main.js:3]: use regions on development stage, but delete it before pushing changes

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,472 @@
...
+
+    this.viewKeyEvents.keyEvent();
+  }
+
+  init() {};
+}
+
+const view = new View();
+
+view.init();
```
Comment (TarasHoliuk) [inline src/scripts/main.js:469]: it seems like you call an empty method

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
your demo link still doesn't work - feel free to ask in the chat if you need help
image

**Pull request closed without merging**